core/src/main/java/io/grpc/DnsNameResolver.java

```diff
@@ -85,8 +94,9 @@
     // Must prepend a "//" to the name when constructing a URI, otherwise it will be treated as an
     // opaque URI, thus the authority and host of the resulted URI would be null.
     URI nameUri = URI.create("//" + name);
-    authority = Preconditions.checkNotNull(nameUri.getAuthority(),
-        "nameUri (%s) doesn't have an authority", nameUri);
+    authority =
+        Preconditions.checkNotNull(
+            nameUri.getAuthority(), "nameUri (%s) doesn't have an authority", nameUri);
```

Comment:
The extra newline for this LOC is unfortunate. (There are much worse cases, but you might claim you should simplify the code in those cases. This case is already pretty simple.)